lib/provider/drift/chat.dart

```diff
@@ -384,6 +440,7 @@ extension _ChatDb on DtoChat {
       ),
       kindIndex: value.kindIndex,
       isHidden: value.isHidden,
+      isArchived: value.isArchived,
       muted: value.muted == null ? null : jsonEncode(value.muted?.toJson()),
```

Comment:
Is seems that you forgot to add `isArchived` flag to `fromDb(ChatRow e)` method (it's a bit higher in this file).